Author: Andreas Beckmann <anbe@debian.org>
Description: fix building with -Werror=implicit-function-declaration

--- a/netcat.c
+++ b/netcat.c
@@ -68,6 +68,7 @@
 #endif
 
 /* includes: */
+#include <time.h>		/* time */
 #include <sys/time.h>		/* timeval, time_t */
 #include <setjmp.h>		/* jmp_buf et al */
 #include <sys/socket.h>		/* basics, SO_ and AF_ defs, sockaddr, ... */
@@ -86,6 +87,8 @@
 #include <unistd.h>
 #endif
 
+void helpme(void);
+
 /* handy stuff: */
 #define SA struct sockaddr	/* socket overgeneralization braindeath */
 #define SAI struct sockaddr_in	/* ... whoever came up with this model */
@@ -1798,7 +1801,7 @@ Debug (("netfd %d from port %d to port %
 #ifdef HAVE_HELP		/* unless we wanna be *really* cryptic */
 /* helpme :
    the obvious */
-helpme()
+void helpme(void)
 {
   o_verbose = 1;
 #ifndef DEBIAN_VERSION
